lib/Parser/RegexCompileTime.cpp

```diff
@@ -3460,6 +3460,12 @@ namespace UnifiedRegex
         innerFollow->UnionInPlace(compiler.ctAllocator, *accumFollow);
         innerFollow->UnionInPlace(compiler.ctAllocator, *body->firstSet);
 
+        if (followSet->IsSingleton())
```

Comment:
I meant to comment on the GetCompactChar line.

---
In reply to: [81023203](https://github.com/Microsoft/ChakraCore/pull/1579#discussion_r81023203) [](ancestors = 81023203)